api.swagger.yaml

```diff
@@ -2974,6 +2974,40 @@ paths:
         - SessionCookieHeader: []
         - APIKeyHeader: []
         - CognitoJwt: [ invotra/group_polls:write ]
+  '/group_polls/{group_poll_id}/vote':
+    parameters:
+      - description: 'The Invotra UUID of the group poll.'
+        in: path
+        name: group_poll_id
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      tags:
+        - 'group polls'
+      description: 'Vote in a group poll.'
+      responses:
+        '201':
+          description: 'The vote in a group poll was successfull.'
```

Comment:
typo - successful

Reply:
in a -> in the?

Reply:
Fixed types, yes the changes to feeds are being done in FT-16 (I put a note in the task desc)